client/src/themes/theme.js

```diff
@@ -3,13 +3,13 @@ import { createMuiTheme } from '@material-ui/core';
 export const theme = createMuiTheme({
   typography: {
     fontFamily: 'Poppins , sans-serif',
-    fontSize: 12,
-    h6: {
-      fontSize: 16,
-    },
-    button: {
-      fontSize: 9,
-    },
+    // fontSize: 12,
```

Comment:
are these not needed anymore?

Reply:
removed these lines

Reply:
Yes removing them is fine. Rhut and I have agreed that he has final say over fontsize.